Modules/context.py

```diff
@@ -94,6 +124,34 @@ def channel(self) -> Optional[str]:
         """
         return self.target if self.bot.is_channel(self.target) else None
 
+    @classmethod
+    async def from_message(cls, bot: 'MechaClient', channel: str, sender: str, message: str):
+        """
+        Creates a context from a IRC message
+
+        Args:
+            bot (MechaClient): MechaClient instance
+            message (str):  raw message
+            sender (str): IRC nickname of the sender
+            channel (str): IRC channel the message was sent from
+
+        Returns:
+            Context
+        """
+        # check if the message has our prefix
+        prefixed = prefix in message
```

Comment:
This will give false positives for messages such as `R@SIGNAL I'm stranded!!!`. In fact, it looks to me like any message including a `!` would be picked up as a command. `messase.startswith(prefix)` should be a better option.